app/services/update_user_compliance_info.rb

```
@@ -54,13 +69,28 @@ def process
-      begin
-        StripeMerchantAccountManager.handle_new_user_compliance_info(new_compliance_info)
-      rescue Stripe::InvalidRequestError => e
-        return { success: false, error_message: "Compliance info update failed with this error: #{e.message.split("Please contact us").first.strip}", error_code: "stripe_error" }
-      end
+      StripeMerchantAccountManager.handle_new_user_compliance_info(new_compliance_info)
     end
 
     { success: true }
   end
+
+  private
```

RuboCop (1.75.5)

[convention] Remove a blank line after `private`.

(Layout/EmptyLinesAroundAccessModifier)